CompetitionTask1/CompetitionTask1/Pages/LoginPage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using ConsoleApp2.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2.Pages
+{
+    public class LoginPage: CommonDriver
+    {
+        
+        public void LoginSteps()
+        {
+            //click on signin button
+            IWebElement signIn = driver.FindElement(By.XPath("//*[@id=\"home\"]/div/div/div[1]/div/a"));
+            signIn.Click();
+
+            // enter username in textbox
+            IWebElement username = driver.FindElement(By.XPath("/html/body/div[2]/div/div/div[1]/div/div[1]/input"));
```

Comment:
Perhaps extract these `XPath` objects into separate fields like you did for `CertifcationPage`.